### PR TITLE
implementing component structure to initialize iframe

### PR DESCRIPTION
**Pull request by ypamplona-flourishfi**

### CODE REVIEW

File: projects/flourish-web-sdk-angular/src/lib/enums/environment.enum.ts [hunk 1]
```diff
@@ -1,5 +1,20 @@
-enum Environment {
+export enum Environment {
```
Comment (ypamplona-flourishfi) [inline projects/flourish-web-sdk-angular/src/lib/enums/environment.enum.ts:1]: I added maximum possibilities to be "stupid client proof", what you guys think?

File: projects/flourish-web-sdk-angular/src/lib/enums/language.enum.ts [hunk 2]
```diff
@@ -1,5 +1,11 @@
-enum Language {
+export enum Language {
```
Comment (ypamplona-flourishfi) [inline projects/flourish-web-sdk-angular/src/lib/enums/language.enum.ts:1]: Same ideia from `environment.enum`

File: projects/flourish-web-sdk-angular/src/lib/enums/environment.enum.ts (hunk 1 above)
Comment (jefersonkrauss) [reply projects/flourish-web-sdk-angular/src/lib/enums/environment.enum.ts:1]: I dislike. It's your enum, the customer should translate them environment to your enum and we should have only one option per environment.

File: projects/flourish-web-sdk-angular/src/lib/enums/language.enum.ts (hunk 2 above)
Comment (jefersonkrauss) [reply projects/flourish-web-sdk-angular/src/lib/enums/language.enum.ts:1]: I dislike. It's your enum, the customer should translate them environment to your enum and we should have only one option per environment.

**Review by jefersonkrauss — CHANGES_REQUESTED** [review]

**Review by jefersonkrauss — APPROVED** [review]

**Pull request merged**